test/cypress/integration/22-collective.edit.test.js

```diff
@@ -111,7 +111,7 @@ describe('edit collective', () => {
     cy.wait(500);
     cy.get('.actions > .btn').click(); // save changes
     cy.get('.backToProfile a').click(); // back to profile
-    const tierCardSelector = '[data-cy="financial-contributions"] [data-cy="contribute-card-tier"]';
+    const tierCardSelector = '[data-cy="financial-contributions"] [data-cy="contribute-card-tier"]:visible';
```

Comment:
This little change is what should fix the test.

The problem comes from the admin cards, that are a dynamic component. They are initially mounted with `display: none` and we then make them visible once the dynamic component is mounted (to make sure we don't display a blank space or a loading spinner). That means that in the DOM, during a short period, we end up with 8 contribution cards (but 4 of them are hidden).